ansible/roles_ocp_workloads/ocp4_workload_integreatly/tasks/post_workload.yml

```diff
@@ -57,19 +74,18 @@
   ignore_errors: True
 
 - name: Create dedicated admin group
-  shell: oc adm groups new dedicated-admins {{ ocp4_workload_integreatly.dedicated_admin_username }}
+  shell: oc adm groups new dedicated-admins {{ ocp4_workload_integreatly_dedicated_admin_username }}
   when: group_output.rc == 1
 
 - name: Add dedicated admin user to dedicated admin group
-  shell: oc adm groups add-users dedicated-admins {{ ocp4_workload_integreatly.dedicated_admin_username }}
+  shell: oc adm groups add-users dedicated-admins {{ ocp4_workload_integreatly_dedicated_admin_username }}
   when: group_output.rc != 1
 
 - name: Print Overview
   debug:
     msg: "{{ item }}"
   with_items:
-    - "user.info: Dedicated Admin User: {{ ocp4_workload_integreatly.dedicated_admin_username }} / {{ _dedicated_admin_password }} (This is the most privileged user account available to consultants and customers in RHMI production/poc environments.)"
-
+    - "user.info: Dedicated Admin User: {{ ocp4_workload_integreatly_dedicated_admin_username }} / {{ ocp4_workload_integreatly_dedicated_admin_user_password }} (This is the most privileged user account available to consultants and customers in RHMI production/poc environments.)"
```

Comment:
@aidenkeating We should probably include more details here. Similar to what we have for 1.x: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/ocp-workload-integreatly/tasks/workload.yml#L55-L62

Can rename "Web App URL" to "Solution Explorer URL" instead.